src/sentry/api/endpoints/project_rules_configuration.py

```diff
@@ -66,6 +71,22 @@ def get(self, request, project):
             elif rule_type.startswith("action/"):
                 action_list.append(context)
 
+        for install in SentryAppInstallation.get_installed_for_org(project.organization_id):
+            _components = SentryAppComponent.objects.filter(
+                sentry_app_id=install.sentry_app_id, type="alert-rule-action"
+            )
+            for component in _components:
+                try:
+                    sentry_app_components.Preparer.run(
+                        component=component, install=install, project=project
+                    )
+                    action_list.append(
+                        serialize(component, request.user, SentryAppAlertRuleActionSerializer())
+                    )
+
+                except APIError:
+                    continue
```

Comment:
Is there a risk associated with this? I don't have the greatest context but I'm wondering what we are expecting these errors to be and whether we should catch them in sentry instead.